autoload/dictionary.vim

```diff
@@ -82,7 +83,7 @@ export def Completor(findstart: number, base: string): any
         return 1
     elseif findstart == 1
         var line = getline('.')->strpart(0, col('.') - 1)
-        var prefix = line->matchstr('\a\+$')
+        var prefix = line->matchstr('\k\+$')
```

Comment:
What does this do? Does your dictionary contain non-alphabetic characters? I just want to understand the specific use-case that makes `\k` valuable.